release/community-stable/oraclelinux/dependabot/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Microsoft Corporation. 
+# Licensed under the MIT License.
```

Comment:
delete this file, this was only needed when we dynamically updated the tag

Reply:
do you mean that when I need to bump up the image version, I will have to add a dockerfile in the "/release/community-stable/oraclelinux/docker" folder?

PS: sorry, I'm learning this for the first time.